### ISSUE TITLE
Volume/Mute Position

### ISSUE THREAD

**Issue by sergiocos** [pix/aeon#77]
Modifying the volume during a movie with subtitles can make you lose some dialogue lines (volume/mute bar overlays the movie in the subtitle area). Maybe that volume/mute bar can be moved on top of the display, or, even better, transformed from horizontal to vertical progressbar and positioned to right.

**Comment by sergiocos** [comment]
a picture to prove my point http://img180.imageshack.us/img180/6678/screenshot000up.jpg

**Comment by pix** [comment]
Closed by 5a87e18819914e9f31178cd78637849a39fa9da3: Volume/Mute Position

  We don't want the volume/mute popup to hide the subtitles:
  so we move the volume/mute dialog to the top when playing
  a video.

```
<animation effect="slide" start="0,0" end="0,-580" time="0"
  condition="VideoPlayer.IsFullscreen">Conditional</animation>
```
